ingeniamotion/configuration.py

```diff
@@ -979,3 +979,27 @@ def change_baudrate(self, baud_rate: CAN_BAUDRATE, servo: str = DEFAULT_SERVO) -
         net.change_baudrate(
             drive.target, baud_rate, vendor_id, prod_code, rev_number, serial_number
         )
+
+    def change_node_id(self, node_id: int, servo: str = DEFAULT_SERVO) -> None:
+        """Change a CANopen device's node ID.
+
+        Args:
+            node_id: New node ID.
+            servo : servo alias to reference it. ``default`` by default.
+
+        Raises:
+            ValueError: If servo is not a CANopen device.
+
+        """
+        drive = self.mc._get_drive(servo)
+        net = self.mc._get_network(servo)
+        if not isinstance(net, CanopenNetwork):
+            raise ValueError(f"Servo {servo} is not a CANopen device.")
+        vendor_id = self.get_vendor_id(servo)
+        (
+            (prod_code, _),
+            (rev_number, _),
+            _,
+            (serial_number, _),
+        ) = self.get_drive_info_coco_moco(servo)
```

Comment:
Use get_product_code, get_revision_number and get_serial_number functions instead get_drive_info_coco_moco to get the drive info. This change will reduce the drive calls

Reply:
Done